src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/TelekineticUtilitySpell.java

```diff
@@ -0,0 +1,116 @@
+package com.mineboundteam.minebound.magic.UtilitySpells;
+
+import com.mineboundteam.minebound.MineBound;
+import com.mineboundteam.minebound.config.IConfig;
+import com.mineboundteam.minebound.item.armor.ArmorTier;
+import com.mineboundteam.minebound.magic.PassiveSpellItem;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.common.ForgeConfigSpec.Builder;
+import net.minecraftforge.common.ForgeConfigSpec.IntValue;
+import net.minecraftforge.event.TickEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.config.ModConfigEvent;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashMap;
+import java.util.List;
+
+
+@Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
+public class TelekineticUtilitySpell extends PassiveSpellItem {
+
+    private final int manaCost;
+
+    static HashMap<ArmorTier, Double> manaCostReductions = new HashMap<>() {{
```

Comment:
Should these be part of the config; I'm fine with the values, but do we want to expose them to the player?

Reply:
We definitely could. I don't see why not. I'll make the change.

Reply:
Done.